engine/launcher/src/main/scala/org/enso/launcher/cli/LauncherApplication.scala

```diff
@@ -99,12 +101,23 @@ object LauncherApplication {
       "Advanced option, use carefully.",
       showInUsage = false
     )
-  private def versionOverride =
+  private def versionOverride = {
     Opts.optionalParameter[SemVer](
       "use-enso-version",
       "VERSION",
       "Override the Enso version that would normally be used."
     )
+  }
+  private def engineLogLevel = {
+    Opts
+      .optionalParameter[LogLevel](
+        "log-level",
+        "LOG-LEVEL",
```

Comment:
Actually now I think that maybe instead of `LEVEL`, I could write `(trace | debug | info | warning | error)` so it will show up in the help text as `--log-level (trace | debug | info | warning | error)`, what do you think ?

Reply:
Might look a bit strange due to line wrapping. I don't mind though.

Reply:
It fits within 80 characters:
![image](https://user-images.githubusercontent.com/1436948/94921111-e6a48500-04b7-11eb-9b0b-84867d444b9e.png)